src/stores/CountryStore.js

```diff
@@ -0,0 +1,37 @@
+//  ### 🤠 ###
+import { defineStore } from 'pinia'
+import { vdbFetchData } from '@/utils/api'
+
+export const useCountryStore = defineStore('CountryStore', {
+  state: () => {
+    return {
+      fetching: false,
+      token: JSON.parse(localStorage.getItem('user-store')).token,
+      countrysPage: null,
```

Comment:
there is no Pagination build in for countries. So you can just call it countries. (suddenly I realize that it should be “countries” and not “countrys” 🙈) I change the route on the backend as well...